openedx/core/djangoapps/user_authn/views/registration_form.py

```diff
@@ -298,13 +298,16 @@ def cleaned_extended_profile(self):
 
 def get_registration_extension_form(*args, **kwargs):
     """
-    Convenience function for getting the custom form set in settings.REGISTRATION_EXTENSION_FORM.
+    Convenience function for getting the custom form set in site configurations or plataform settings
+    REGISTRATION_EXTENSION_FORM.
 
-    An example form app for this can be found at http://github.com/open-craft/custom-form-app
+    Documentation on how to enable this feature can be found on
+    https://github.com/Pearson-Advance/pearson-core/blob/master/docs/pearson_core_features.md#amazon-custom-registration-form
     """
-    if not getattr(settings, 'REGISTRATION_EXTENSION_FORM', None):
-        return None
-    module, klass = settings.REGISTRATION_EXTENSION_FORM.rsplit('.', 1)
+    # No Default valued is passed since we prioritize site configurations over site settings.
+    registration_extra_form = configuration_helpers.get_value('REGISTRATION_EXTENSION_FORM',
```

Comment:
You should keep checking the existence of the default value, as it's now, it will throw an error if the setting is not present. Also if the setting is present but is None, it will fail on line 310. @Nekenhei

Reply:
Hi @Squirrel18 you're right, I've removed by error an if statement that handles the situation when none of the sources has the setting set. 

In the original code, the default value was none, so I changed to firstly check for site conf, then plataform settings and if none of them are presented, return None as it's working on master.